cypress/integration/loans/loans-claim-returned.spec.js

```diff
@@ -0,0 +1,177 @@
+import moment from 'moment';
+import uuid from 'uuid';
+import devTeams from '../../support/dictionary/devTeams';
+import permissions from '../../support/dictionary/permissions';
+import TopMenu from '../../support/fragments/topMenu';
+import TestTypes from '../../support/dictionary/testTypes';
+import CheckInActions from '../../support/fragments/check-in-actions/checkInActions';
```

Comment:
'CheckInActions' is defined but never used

Reply:
Done.